module_utils/network_lsr/argument_validator.py

```diff
@@ -364,9 +364,9 @@ def _validate_impl(self, value, name):
             if validator.required:
                 raise ValidationError(name, "missing required key '%s'" % (setting))
             if not self.all_missing_during_validate:
-                default_value = validator.get_default_value()
-                if default_value is not ArgValidator.MISSING:
-                    result[setting] = default_value
+                v = validator.get_default_value()
+                if v is not ArgValidator.MISSING:
+                    result[setting] = v
```

Comment:
Especially in this case using "v" for "default_value" when there is also a validator is messing with my brain since it is assuming that "v" means "validator" here after this change. If you dislike "default_value" here, then "default" might be a good alternative.

Reply:
There are other short variables like `k`, `v`, `kk`. As you might guess, I don't have the distaste for them, if they are  temporary variable, that is used in a very narrow context (spanning very few lines of code).

I would prefer `val`, as `default` sounds like a reserved keyword (which it isn't, in Python). But I don't really mind. Renamed to `default`!!